scripts/auto_release/PythonSdkLiveTest.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           --user "anything:$(USR_TOKEN)" \
           -d "{\
                   \"title\":\"[AutoRelease] $new_branch(Do not merge)\",\
-                  \"body\":\"$test_result Please Add the link issue \",\
+                  \"body\":\"$test_result $(ISSUE_LINK) \",\
                   \"head\":\"$(USR_NAME):$new_branch\",\
```

Comment:
add validation to confirm `ISSUE_LINK` and `BASE_BRANCH` is not null.
![image](https://user-images.githubusercontent.com/70930885/127621331-994b4451-c6d9-4d16-95fc-5ffe5d3e68e6.png)